keyboards/nullbitsco/nibble/keymaps/via/keymap.c

```diff
@@ -142,11 +143,20 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
       break;
 
     default:
-    break;    
+    break;
   }
 return true;
 }
 
+void encoder_update_kb(uint8_t index, bool clockwise) {
```

Comment:
Looks like the other keymaps also suffer from this issue

```suggestion
void encoder_update_user(uint8_t index, bool clockwise) {
```